isis/scripts/zenodo_to_authors.py

```diff
@@ -0,0 +1,50 @@
+# Uses the .zenodo.json file to generate an AUTHORS.rst file. 
+# No arguments are necessary if $ISISROOT is set.
+
+import os
+import argparse
+import json
+
+parser = argparse.ArgumentParser(description="Generate an AUTHORS.rst file from a .zenodo.json file")
+parser.add_argument("zenodo_input_file", nargs='?', type=str, 
+                    help="The file path to the .zenodo.json file.", default="$ISISROOT/../.zenodo.json")
+parser.add_argument("authors_output_file", nargs='?', type=str, 
+                    help="The path for the output AUTHORS.rst file.", default="$ISISROOT/../AUTHORS.rst")
+
+args = parser.parse_args()
+
+zenodo_path = args.zenodo_input_file
+authors_path = args.authors_output_file
+
+# Files to open
+if (("$ISISROOT" in args.zenodo_input_file) or ("$ISISROOT" in args.authors_output_file)):
+  if ("ISISROOT" in os.environ):
+    zenodo_path = os.path.expandvars(args.zenodo_input_file)
+    authors_path = os.path.expandvars(args.authors_output_file)
+  else:
+    print("Please set $ISISROOT and re-run this script or specify the input and output files.")
+    exit()
```

Comment:
I do not like setting the defaults and then having to do this; especially,  because this could error. My rule of thumb is that a script with default arguments should be able to just run; if I have to set environment variables to run it with default arguments I may as well just pass the arguments. I would just make the arguments required.

Reply:
Can do. My initial thought was that the default use of this script would be automated (in the future!) rather than normally being something an individual user would run, so I was thinking "$ISISROOT" is set would be the default case. I see your point, though, and am happy to just remove the defaults and make the arguments required.